.ci/install_kata_image.sh

```diff
@@ -45,6 +45,7 @@ build_rust_image() {
 	case "$build_method" in
 		"distro")
 			distro="${osbuilder_distro:-fedora}"
+			OS_VERSION="30"
```

Comment:
Could you try switching to a more recent version of Fedora (the currently released one is 35 now)

Reply:
@c3d - I'm currently looking into it, but unfourtunately with version 34 of fedora the containerd tests fail, but they pass with fedora 30. I'm hoping to end up with tests working on fedora 34, but that's proving a challenge at the moment.